src/foam/u2/view/CardSelectionView.js

```diff
@@ -97,6 +100,11 @@ foam.CLASS({
                 cardSelectViewConfig.data$ = valueSimpSlot;
                 cardSelectViewConfig.data  = valueSimpSlot.get();
 
+                // Set choice selection based on predicate
+                if ( self.selectionPredicate && self.selectionPredicate.f(choice) ) {
```

Comment:
Wouldnt this override the existing self.data if someone populates that property directly?

Reply:
Fixed in https://github.com/kgrgreer/foam3/pull/2410/commits/bd25ca5a53f5acb87d957d75b09fae6c4d58e194.